src/backend/cgcv.c

```diff
@@ -2162,6 +2162,10 @@ unsigned cv4_typidx(type *t)
 #endif
 #if MARS
         case TYref:
+        case TYnref:
+#if TARGET_SEGMENTED
+        case TYfref:
```

Comment:
You can remove all the `TYfref`s. They are never going to happen, as D is never going to support 16 bit segmented memory models.